### DISCUSSION TITLE
Access control for some areas of the reports?

### DISCUSSION THREAD

**Discussion by JSXRED** [evidence-dev/evidence#2558] [General]
is there a plan or are there already preparations for access control? as we are big fans, we are planning to expand evidence further, but as in every larger structure, there is a need somewhere to make some smaller sections not necessarily accessible to everyone.

so as an idea, an SSO system like keycloak or similar would be available and would provide evidence with sufficient information on who the current user is. 

As a wishful thinking, the menu would then remove 1-2 links to which the user has no access and also make 1-2 URL locations inaccessible. The latter could also be solved with a reverse proxy, like nginx, and check with auth_request whether the user is allowed to access it. however, it would be nicer if the menu entries were not there at all.

i look forward to your thoughts and arguments against it ;).
JSX

**Comment by marvinhorst** [comment]
Just came across this project (evidence.dev) and would definitely look into it further if access control was part of the project. This is a must have for us. We're currently using Pentaho CE 9.4 just for the CDE dashboard module. We like it's features and really like it's flexibility to build any custom component you want but this module is no longer being maintained.
